examples/plot_clickable_image.py

```diff
@@ -0,0 +1,70 @@
+"""
+=========================================
+Demonstration of how to use ClickableImage / generate_2d_layout.
+=========================================
+
+In this example, we open an image file, then use ClickableImage to
+return 2D locations of mouse clicks (or load a file already created).
+Then, we use generate_2d_layout to turn those xy positions into a layout
+for use with plotting topo maps. In this way, you can take arbitrary xy
+positions and turn them into a plottable layout.
+"""
+# Authors: Christopher Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+import sys
+sys.path.insert(0, '../')
+from scipy.ndimage import imread
+import numpy as np
+from matplotlib import pyplot as plt
+import mne
+from mne.viz.utils import ClickableImage, add_background_image
+from mne.channels.layout import generate_2d_layout
+from os import path as op
+
+print(__doc__)
+
+# Set parameters and paths
+plt.rcParams['image.cmap'] = 'gray'
+im_path = op.join(op.dirname(__file__), '..', 'mne', 'data',
+                  'image', 'mni_brain.gif')
+
+# We've already clicked and exported
+layout_path = op.join(op.dirname(__file__), '..', 'mne', 'data', 'image', '')
```

Comment:
you shouldn't need the trailing `''`, if you do something in the underlying code is probably wrong (i.e., not using `os.path`)...